tests/cross/src/test/scala/tests/pc/CompletionDocSuite.scala

```diff
@@ -200,15 +210,25 @@ class CompletionDocSuite extends BaseCompletionSuite {
       |}
     """.stripMargin,
     s"""
-       |$predefDocString
+       |${predefDocString(commonlyUsedTypesPre2134)}
        |Predef scala
        |DeprecatedPredef scala
        |""".stripMargin,
     includeDocs = true,
     compat = Map(
-      "2.13" ->
+      "2.13.2" ->
+        s"""
+           |${predefDocString(commonlyUsedTypesPre2134)}
+           |Predef scala
+           |""".stripMargin,
+      "2.13.3" ->
+        s"""
+           |${predefDocString(commonlyUsedTypesPre2134)}
+           |Predef scala
+           |""".stripMargin,
+      "2.13.4" ->
```

Comment:
If "2.13.2", "2.13.3", and "2.13.4" are all the same can't this just stay "2.13" instead of separating them out?

Reply:
There are actually different, 2.13.4 has changed docs. Might not be that noticable with the naming :sweat_smile:

Reply:
`commonlyUsedTypesPre2134` and `commonlyUsedTypesPost2134` Post/Pre

Any idea on naming? This one is pretty bad I guess.

Reply:
Honestly, I don't think it's that bad. I just _totally_ glossed over the `Post/Pre` part. Do you think it's clearer if you preface it with `post1234`/`pre1234` instead of the suffix? /shrug